assets/src/dashboard/app/views/templates/detail.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useEffect, useState, useContext } from 'react';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import useRouteHistory from '../../router/useRouteHistory';
+import { ApiContext } from '../../api/apiProvider';
+import { TransformProvider } from '../../../../edit-story/components/transform';
+import FontProvider from '../../font/fontProvider';
+import { CardGallery, PreviewPage } from '../../../components';
+
+const PageContainer = styled.div`
+  display: flex;
+  padding: 20px;
```

Comment:
we have `theme.pageGutter.desktop` if it fits your use case here for consistency with page spacing.

Reply:
Ooo, I'll take a look.  I'm going to do a deep dive into `detail.js` in my next PR (when I added in the template information and stuff), so I think the css for this page might change a little, but I'll keep that theming variable (and be sure to reference the theme in general) to get the styles right.